doc/HOWTO_RELEASE.rst.txt

```diff
@@ -113,7 +119,7 @@ Wine
 For building Windows binaries on OS X Wine can be used. In Wine the following
 needs to be installed:
 
-* Python 2.6-2.7 and 3.2
+* Python 2.6-2.7 and 3.3
```

Comment:
Note that numpy-vendor currently produces binaries for 2.7, 3.3, 3.4. We should add 3.5 and Carl's work at some point.